data-mermaid/mermaid-webapp
# Melissa/m646 fix sites mr sort

Steps to test:

GO to each of the sites and MRs page
click Export X
Expect that the resultant csv has items sorted by name
